packages/itwin/grouping-mapping-widget/src/components/Groups/GroupsVisualization.tsx

```diff
@@ -33,6 +33,8 @@ export interface GroupsVisualizationProps extends GroupsProps {
   emphasizeElements?: boolean;
   hideVisualizationToggle?: boolean;
   hideRefreshIcon?: boolean;
+  deleteConfirmationMessage?: (group: GroupMinimal) => JSX.Element;
```

Comment:
Seems a bit weird that `deleteConfirmationMessage` is a function that returns an element. I would name it `deleteConfirmationContentFactory`.